js/researches/german/passivevoice-german/splitSentences.js

```diff
@@ -0,0 +1,52 @@
+var getSentences = require( "../../../stringProcessing/getSentences.js" );
+var stopwords = require( "../../german/passivevoice-german/stopwords.js" )();
+var arrayToRegex = require( "../../../stringProcessing/createRegexFromArray.js" );
+var stripSpaces = require( "../../../stringProcessing/stripSpaces.js" );
+
+var forEach = require( "lodash/forEach" );
+var isEmpty = require( "lodash/isEmpty" );
+
+var stopwordRegex = arrayToRegex( stopwords );
+
+/**
+ * Splits sentences into subsentences based on stopwords.
+ *
+ * @param {string} sentence The sentence to split.
+ * @param {Array} matches The array with matched stopwords.
+ * @returns {Array} The array with subsentences.
+ */
+function splitOnWord( sentence, matches ) {
```

Comment:
At the moment not, since we need to use a different approach in English. There might be languages where we can use a similar or the same function, but for now there isn't 1 universal function that will work everywhere.